api_spec.yaml

```diff
@@ -421,16 +418,13 @@ paths:
                       status:
                         type: string
                         enum: ['UNKNOWN_AUTH_CODE_ERROR',
-                          'AUTH_CODE_EXPIRY_ERROR',
                           'UNKNOWN_CLIENT_SECRET_ERROR', 
                           'UNKNOWN_CLIENT_ID_ERROR',
-                          'REFRESH_TOKEN_EXPIRY_ERROR',
-                          'REDIRECT_URI_MISMATCH_ERROR',
+                          'UNKNOWN_REDIRECT_URI_ERROR',
                           'UNKNOWN_REFRESH_TOKEN_ERROR',
                           'UNKNOWN_SCOPE_ERROR',
```

Comment:
the flow diagram doesn't seem to actually ever throw UNKNOWN_SCOPE_ERROR. We can just remove this from the output.

Reply:
its currently being used in `client_credential` flow , if the scope passed by the user is not a part of `client_allowed_scope` then an UNKNOWN_SCOPE_ERROR is thrown.